feature/challenge-reward/data/src/main/java/com/appcoins/wallet/feature/challengereward/data/ChallengeRewardManager.kt

```diff
@@ -0,0 +1,36 @@
+package com.appcoins.wallet.feature.challengereward.data
+
+import android.app.Activity
+import com.appcoins.wallet.feature.challengereward.data.model.WalletChallengeRewardListener
+import com.fyber.fairbid.ads.OfferWall
+import com.fyber.fairbid.ads.offerwall.ShowOptions
+
+object ChallengeRewardManager {
+  private const val APP_ID = "135904"
```

Comment:
I think it would be better to put this as an environment variable! I believe we will need an appId for PRD as well